swiftinterfaceparser/SwiftInterface.g4

```diff
@@ -496,7 +496,7 @@ requirement_list : requirement (OpComma requirement)*  ;
 requirement : conformance_requirement | same_type_requirement  ;
 
 conformance_requirement : type_identifier ':' type_identifier | type_identifier ':' protocol_composition_type  ;
-same_type_requirement : type_identifier OpEqEq type  ;
+same_type_requirement : type_identifier operator type  ;
```

Comment:
is this related to the PR ?

Reply:
Only in that I got hip deep into the PR and would have failed all the unit tests that were unignored (see #3 above). I was tempted to put it into a separate PR, but it was a single word change...